tests/build.go

```diff
@@ -116,6 +116,15 @@ func Build(o *option.Option) {
 				gomega.Expect(stdErr).Should(gomega.ContainSubstring("progress flag set:2"))
 			})
 
+			ginkgo.It("build image with --output flag", func() {
+				outputFilePath := filepath.Join(buildContext, "out.tar")
+				dest := fmt.Sprintf("type=tar,dest=%s", outputFilePath)
```

Comment:
Do you know where `type=tar` is mentioned? I did a global search in the nerdctl repo for all the `*.go` files and could find one.

Reply:
It's from this command [reference](https://github.com/containerd/nerdctl/tree/v0.23.0#whale-nerdctl-build) doc. I think it's a docker compatible flag supported by nerdctl.